### ISSUE TITLE
[BUG] Language audio does't change ? 

### ISSUE THREAD

**Issue by lavara123** [jhomlala/betterplayer#579]
**Describe the bug**
Test url (hls file):
[https://facebcdn.xyz/hls2/02/00006/,jbib9xgelmjy_n,.urlset/master.m3u8](https://facebcdn.xyz/hls2/02/00006/,jbib9xgelmjy_n,.urlset/master.m3u8) 

```dart
    BetterPlayerDataSource betterPlayerDataSource = BetterPlayerDataSource(
      BetterPlayerDataSourceType.network,
      "https://facebcdn.xyz/hls2/02/00006/,jbib9xgelmjy_n,.urlset/master.m3u8",
      videoFormat: BetterPlayerVideoFormat.hls,
      headers: {
        "Referer":
            "https://facebcdn.xyz/",
        "User-Agent":
            "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:59.0) Gecko/20100101 Firefox/59.0"
      },
    );
```

**Expected behavior**
I can change the audio. 

**Additional context**
I guess it doesn't change because the audio file extension is "txt"

You can test it here, the file works fine:
[https://demo.theoplayer.com/test-your-stream-with-statistics](https://demo.theoplayer.com/test-your-stream-with-statistics)

**Comment by LastxTemplar** [comment]
Please provide a workable piece of code

**Comment by lavara123** [comment]
> Please provide a workable piece of code

I added the code

**Comment by LastxTemplar** [comment]
> > Please provide a workable piece of code
> 
> I added the code

Where are you trying to change the audio in that code?

**Comment by lavara123** [comment]
> > > Please provide a workable piece of code
> > 
> > 
> > I added the code
> 
> Where are you trying to change the audio in that code?

Sound list comes automatically in better_player, you just need to select it
![Screenshot_20210629-235844490 (1)](https://user-images.githubusercontent.com/12005386/123866817-3c6a6880-d936-11eb-9646-db24134295ca.jpg)

**Comment by LastxTemplar** [comment]
Yeah, I can see the 2 audio tracks, if I play it in VLC it shows three of them.
If you try to send a request to the url you get this:
![Image](https://i.imgur.com/7obtRo8.png)

I'm not expert by any means but it looks to me like only two tracks are defined there, the ones starting with
`#EXT-X-MEDIA:TYPE=AUDIO`

Maybe the third one is part of the actual video or something.

**Comment by jhomlala** [comment]
This issue is connected with https://github.com/google/ExoPlayer/issues/6101. However I will be able to provide fix for that with next release.

**Comment by lavara123** [comment]
> This issue is connected with [google/ExoPlayer#6101](https://github.com/google/ExoPlayer/issues/6101). However I will be able to provide fix for that with next release.

Thanks, I'm looking forward to it 👍

**Comment by jhomlala** [comment]
Fixed in 0.0.71.
